src/utils/special-cases.ts

```diff
@@ -82,6 +82,17 @@ const specialCases: Record<string, (o: SpecialCaseOpts) => void> = {
       emitAssetDirectory(resolve(dirname(id), '..', 'bin'));
     }
   },
+  'remark-prism' ({ id, emitAsset }) {
+    const file = 'remark-prism/src/highlight.js';
+    if (id.endsWith(file)) {
+      try {
+        const prefix = id.slice(0, -file.length);
+        emitAsset(resolve(prefix, `prismjs/components/prism-markdown.js`));
```

Comment:
Should this only be adding `prism-markdown` or should we be including all of the files in the components folder here since we won't know which are needed?

Reply:
Yeah good point! Added in 43d6a45 

I also added an integration test in 85b7608